lib/abstract_importer/base.rb

```diff
@@ -175,9 +176,8 @@ def map_foreign_key(legacy_id, plural, foreign_key, depends_on)
       return nil if legacy_id.nil?
       return legacy_id unless use_id_map_for?(depends_on)
       id_map.apply!(depends_on, legacy_id)
-    rescue IdMap::IdNotMappedError
-      record_no_id_in_map_error(legacy_id, plural, foreign_key, depends_on)
-      nil
+    rescue KeyError
+      raise IdNotMappedError, "#{plural}.#{foreign_key} will be nil: a #{depends_on.to_s.singularize} with the legacy id #{legacy_id} was not mapped."
```

Comment:
So this will bust any import where a foreign key doesn't map to a record? I know for Members that happens from time to time.